spec/registrant_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    @registrant_2 = Registrant.new('Penny', 15 )
+  end
+  
+  describe '#initialize' do
+    it 'can initialize' do
+      expect(@registrant_1).to be_an_instance_of(Registrant)
+      expect(@registrant_1.name).to eq('Bruce')
+      expect(@registrant_1.age).to eq(18)
+      expect(@registrant_1.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+    end
+  end
+
+  describe '#permits' do
+    it 'has a permit? method' do
+      expect(@registrant_1.permit?).to eq(true)
+    end
+
+    it 'has an earn permit method' do
+      @registrant_2.earn_permit
```

Comment:
I would include both registrants in this test as well because then you can truly check that your `earn_permit` method is doing what you expect. It should not change `permit` unless they are the correct age.